code/workspaces/web-app/package.json

```diff
@@ -3,8 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "auth0-js": "^9.13.2",
     "@material-ui/core": "^4.9.0",
+    "@material-ui/icons": "^4.9.1",
```

Comment:
I do think this is clearer- should we be using this approach everywhere rather than just getting icons via the call in public/index.html, or is there another difference?

Reply:
That's a good question @iwalmsley. Based on the information at 
https://material-ui.com/components/icons/#font-vs-svg-which-approach-to-use it seems like the svg icons package is the preffered solution. 

If I remember correctly, some of the code requires the icon name to be passed into it as a string to render the right icon so this would need to be updated if we wanted to change over to the new solution. I don't know whether this is something worth doing though!